### DISCUSSION TITLE
Failed to run sql query: function "name of function" does not exist

### DISCUSSION THREAD

**Discussion by andrewjhop** [supabase/supabase#12172] [Questions]
Anyone else having issues creating functions?
CREATE FUNCTION, returns 'success', and the function appears in Database > Functions, yet trying to query the function, or access via Bash, returns that error (i.e. does not exist).

I've created several functions now, all with no found syntax errors, and all return the same error.

Have refreshed schema cache with: NOTIFY pgrst, 'reload schema' - but no luck.

This problem only started happening yesterday. Functions made previous to yesterday (with almost identical code) are working.

Any ideas? :/

**Comment by GaryAustin1** [comment]
You probably need to provide more info and/or show your function and call.  I have an instance that I paused/restored this morning (so has latest code) and no issues with new functions.
